try {
	fn();
} catch {
	console.log("error");
}
/**************************************************/
try{fn();}catch{console.log("error");}
/**************************************************/
try
{
	fn(  );
} catch
{
	console.log( "error" );
}